aioxmpp/pubsub/service.py

```diff
@@ -537,7 +537,8 @@ def publish(self, jid, node, payload, *, id_=None):
         the node, it is replaced. If no ID is given, a ID is generated by the
         server.
 
-        Return the ID of the item as published.
+        Return the ID of the item as published (or :data:`None` if the server
+        does not inform us).
```

Comment:
I would propose to do that in another PR and update *all* the docstrings, so that the documentation of the pubsub module has a consistent style.